paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetBillingConfigurationTest.kt

```diff
@@ -239,6 +239,13 @@ internal class PaymentSheetBillingConfigurationTest {
         integrationType = integrationType,
         resultCallback = ::assertCompleted,
     ) { testContext ->
+        networkRule.enqueue(
+            method("POST"),
+            path("/v1/consumers/sessions/lookup"),
+        ) { response ->
+            response.setResponseCode(500)
+        }
+
```

Comment:
We do this on all other tests so that Link UI (2FA modal) does not interfere in the UI.